src/web/client/dal/remoteSaveProvider.ts

```diff
@@ -170,7 +175,9 @@ async function saveDataToDataverse(
                     entityName,
                     httpMethod.PATCH,
                     new Date().getTime() - requestSentAtTime,
-                    JSON.stringify(response)
+                    JSON.stringify(response),
+                    fileExtensionType,
+                    telemetryEventNames.WEB_EXTENSION_SAVE_FILE_FAILED
```

Comment:
same here - the name could be dataverse save specific. Save of a file is "Save to VFS + Save to Dataverse" = success scenario

Reply:
done